_pages/other.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Others
+layout: page
+sidenav: false
+permalink: /others/
```

Comment:
I think this should changed to singular or update the filename and the child array to match this one.